README.rst

```diff
@@ -137,6 +181,27 @@ Here is the quickest way to build via a cmd/bash shell
   # To edit TeleSculptor code, open the
   # build/external/telesculptor-build/TeleSculptor.sln
 
+This will produce an installer file which can install TeleSculptor in the same
+way as the downloadable installer, as detailed in the next section.
+
+Installation
```

Comment:
I'm wondering if this section should go after all the build options (before MAP-Tk Tools).  As it is, we talk about installation and then go back to talking about build options, which seems odd.  

Maybe an even better option is to put the installation section before the `Build TeleSculptor` section.  Since I expect most users would just download and install this.

Reply:
I agree that it's currently in a weird place and think it could go before build. However, in that case, I don't think `Overview of Directories` makes sense because I don't think many end users will care about that. I'm messing around with other places to put it now, in addition to some formatting issue from my incompetence at reStructuredText.

Reply:
It's moved. I was pretty heavy handed in pointing build-from-source users to the right sections so I don't think there should be any confusion there.

Reply:
As a separate commmit I moved `Overview of Directories`. I don't think that's critical but I do think it reads better like that.